csv2wiki/__init__.py

```diff
@@ -119,67 +119,83 @@
-                           spaces) are processed similarly, with
-                           a single space being emitted between
-                           their content in the output.
-
-                           Everything after the number(s) is ignored.
-                           Typically, one puts a comment there
-                           describing the columns that correspond to
-                           the numbers.  While there is no requirement
-                           to begin the comment with "#", it is
-                           a good idea to do so, because that provides
-                           another way to distinguish section-content
-                           lines from section-header lines, which can
-                           greatly help readability in a complex sec_map.
+                         - If it begins with a pipe character ("|"),
+                           it indidcates a text line where any instance
+                           of "{N}" (N is an integer) in that text line
+                           will be replaced with the row data for that
+                           column.
+
+                           If N is also the cat_col, then it will also
+                           be a wiki link to the category.
+
+                           csv2wiki used to support sec_map lines that
+                           had only integers in them, but in order to
+                           fulfill that functionality, you should do:
+
+                           | {N}
+
+                         - If it begines with a pound sign ("#"), it will
+                           be discarded from the sec_map as a comment.
 
                        Here is an example 'sec_map':
 
                          sec_map:  .   Applicant {1}
-                                   1
+                                   # A comment about applicants
+                                   | {1
+                                   .   Contents
+                                   | __TOC__
                                    .   Proposal
                                    ..  Executive Summary
-                                   12 
+                                   | {12}
                                    ..  Detailed Proposal
-                                   15   
+                                   | {15}
                                    .   Organization Info
-                                   21 19 20  # Title, First Name, Last Name
-                                   21        # Title
-                                   22        # Phone
-                                   23        # Email
-                                   29        # City
-                                   30        # State / Province
-                                   31        # Postal Code
-                                   32        # Country
+                                   | Title, First, Last: {21} {19} {20}
```

Comment:
The example here is odd, because you give the Title again after already having given it.  It's syntactically correct, sure, but it might confuse people.

Reply:
I really just replaced the comments with text as an example.  Before, it was:
`21 19 20  # Title, First Name, Last Name`
on that line.  Was definitely going for quick and easy examples.  Want me to spend more time and come up with a better option?

Reply:
Nah, don't worry about it now.  I might do a holistic pass over those docs later, but for now no need to worry about it.